migrations/1758181225.sh

```diff
@@ -0,0 +1,21 @@
+echo "Update Docker daemon configuration with MTU optimization for Tailscale/WireGuard VPNs and improved ulimits"
+
+# Only update if Docker is installed
+if command -v docker >/dev/null 2>&1; then
```

Comment:
Should prob update the config if it's present, but only re-run the config script if Docker is installed.